app/models/user.rb

```diff
@@ -1169,8 +1169,11 @@ def serializable_hash(options = nil)
     json
   end
 
+  def personal_records
```

Comment:
It's dangerous to call this `personal_records` and then actually return WCIF.
In the world of Ruby object models, this should actually return the `.compact.flatten`ed array. The consumer of the method should then map it to WCIF if necessary.

Reply:
Changed